src/_P018_Dust.ino

```diff
@@ -58,6 +59,17 @@ boolean Plugin_018(uint8_t function, struct EventStruct *event, String& string)
       break;
     }
 
+    # if FEATURE_MQTT_DISCOVER
+    case PLUGIN_GET_DISCOVERY_VTYPES:
+    {
+      for (uint8_t i = 0; i < event->Par5; ++i) {
+        event->ParN[i] = static_cast<int>(Sensor_VType::SENSOR_TYPE_DUSTPM2_5_ONLY);
```

Comment:
Typo `ParN` ?

Reply:
New 'feature', it's a `union` with `Par1`..`Par5`, to be able to set these in a for loop.

Reply:
@tonhuisman maybe same for strings?